### PR TITLE
add new release note for v1.11

### PR DESCRIPTION
**Pull request by zhzhuang-zju**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
add new release note for v1.11

**Which issue(s) this PR fixes**:
Parts of  #5466

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by zhzhuang-zju** [comment]
assign @RainbowMango 

The sections `What's New` and `Contributors` will be added later

File: docs/CHANGELOG/CHANGELOG-1.11.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+## Downloads for v1.11.0
+
+Download v1.11.0 in the [v1.11.0 release page](https://github.com/karmada-io/karmada/releases/tag/v1.11.0).
+
+## What's New
+
+## Other Notable Changes
+### API Changes
+- Introduced `Suspension` to the `PropagationPolicy/ClusterPropagationPolicy` API to provide a cluster-level resource propagation pause and resume capabilities. ([#4838](https://github.com/karmada-io/karmada/pull/4838), @a7i)
+- Introduced `Dispatching` condition to represent the dispatching status. ([#5317](https://github.com/karmada-io/karmada/pull/5317), @a7i)
```
Comment (RainbowMango) [inline docs/CHANGELOG/CHANGELOG-1.11.md:11]:
```suggestion
- Introduced `Dispatching` condition to the `Work` API to represent the dispatching status. ([#5317](https://github.com/karmada-io/karmada/pull/5317), @a7i)
```
By the way, I also updated the release notes on the PR too.

File: docs/CHANGELOG/CHANGELOG-1.11.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+- In this release, Karmada has invested significant effort in enhancing Karmada maturity based on [Clomonitor check sets](https://clomonitor.io/docs/topics/checks/). So far, we have achieved a score of [99](https://clomonitor.io/projects/cncf/karmada), and the last check (Signed releases) will be passed after 5 releases. This indicates that the Karmada community has made great strides in both security and maturity. (contributors: @zhzhuang-zju, @B1F030, @adiya7302, @Akash-Singh04, @RainbowMango)
+- Introduced [SBOM](https://www.aquasec.com/cloud-native-academy/supply-chain-security/sbom/) to release assests to enhance transparency around Karmada components and dependencies, bolster the security posture of our project. The composition and usage of SBOM can refer to [SBOM DOC](https://karmada.io/docs/next/administrator/security/verify-artifacts#sbom). ([#5110](https://github.com/karmada-io/karmada/pull/5110), @zhzhuang-zju)
+- Introduced [SLSA provenance file](https://slsa.dev/spec/v0.1/) to release assets, with which Karmada users verify the artifacts to prevent counterfeiting. ([#5178](https://github.com/karmada-io/karmada/pull/5178), @zhzhuang-zju)
+
+### Features & Enhancements
```
Comment (RainbowMango) [inline docs/CHANGELOG/CHANGELOG-1.11.md:68]: Please group the notes by components. It makes it easier for people to find changes they are interested in.

File: docs/CHANGELOG/CHANGELOG-1.11.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+
+### `generation` and `observedGeneration` fields for federated workload resources are aligned with the semantics of a single Kubernetes cluster
+
+The `observedGeneration` in the workload status at the federation level will only be set to the `.metadata.generation` value when the `status.observedGeneration` of the resources distributed to all member clusters is greater than or equal to `.metadata.generation`. 
+
+This ensures that the corresponding controllers in each member cluster have completed processing of the resource. This allows for correct conditional judgments in accordance with the habits of Kubernetes clusters, such as applying the condition `status.observedGeneration >= .metadata.generation` during the publishing process of the workload resources.
+
+For a detailed description of this feature, see the [issue](https://github.com/karmada-io/karmada/issues/4870).
+
+(Feature contributors: @yike21)
```
Comment (XiShanYongYe-Chang) [inline docs/CHANGELOG/CHANGELOG-1.11.md:79]: Can add more contributors.

File: docs/CHANGELOG/CHANGELOG-1.11.md [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+
+- More detailed command output information
+
+The output of the `karmadactl get cluster` command now add the information of `Zones`, `Region`, `Provider`, `API-Endpoint`, and `Proxy-URL`.
+
+With these capability enhancements, the operational experience of `karmadactl` could be improved. The new capabilities and more information about `karmadactl` can be obtained using `karmadactl --help`.
+
+(Feature contributors: @hulizhe, @zhzhuang-zju, @whitewindmills, @a7i, @guozheng-shen, @grosser)
+
+### `generation` and `observedGeneration` fields for federated workload resources are aligned with the semantics of a single Kubernetes cluster
```
Comment (RainbowMango) [inline docs/CHANGELOG/CHANGELOG-1.11.md:71]:
```suggestion
### Consistent generation semantics for multi-cluster workloads
```

File: docs/CHANGELOG/CHANGELOG-1.11.md [hunk 5]
```diff
@@ -0,0 +1,257 @@
...
+
+The `observedGeneration` in the workload status at the federation level will only be set to the `.metadata.generation` value when the `status.observedGeneration` of the resources distributed to all member clusters is greater than or equal to `.metadata.generation`. 
+
+This ensures that the corresponding controllers in each member cluster have completed processing of the resource. This allows for correct conditional judgments in accordance with the habits of Kubernetes clusters, such as applying the condition `status.observedGeneration >= .metadata.generation` during the publishing process of the workload resources.
```
Comment (RainbowMango) [inline docs/CHANGELOG/CHANGELOG-1.11.md:75]:
```suggestion
In this release, Karmada has introduced consistent generation semantics for workloads running on multiple clusters. This update provides a reliable reference for release systems, enhancing the precision of multi-cluster deployments. By standardizing the generation semantics, Karmada simplifies the release process and ensures that workload statuses are consistently tracked, making it easier to manage and monitor applications across multiple clusters.
```

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**